Diagnostics/Get-NetView.PS1

```diff
@@ -1596,7 +1641,7 @@ function EnvCreate {
     Try {
         New-Item -ItemType directory -Path $OutDir -ErrorAction Stop | Out-Null
     } Catch {
-        Throw ("Get-NetView: Failed to create directory '$OutDir' because " + $error[0])
+        Throw ("Get-NetView : Failed to create directory '$OutDir' because " + $error[0])
```

Comment:
Space added to be consistent with other Cmdlets on exception thrown (e.g. Set-VM)